src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +522,35 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if is_fusion_transcript_segment:
+            # Check if is_fusion_transcript_segment is given
+            mane_transcripts = self.mane_transcript_mappings.get_gene_mane_data(gene)
```

Comment:
What if there is no MANE transcripts for a gene? You should test these cases

Reply:
I do not see any tests where the `gene` is set to `None`. Please add tests for these cases. It will likely affect the later call `self.uta_db.get_transcripts(gene=gene, alt_ac=alt_ac)`

Reply:
I would also add tests for when `strand` is `None`